### ISSUE TITLE
argument void has no protection

### ISSUE THREAD

**Issue by skoppe** [libmir/mir-core#38]
```
import mir.algebraic;
struct Test {
  alias Value = void;
}

alias B = Nullable!Test;
```

```
./mir-core-1.1.59/mir-core/source/mir/reflection.d(741,27): Error: argument void has no protection
./dlang/dmd-2.094.2/linux/bin64/../../src/phobos/std/meta.d(1381,45): Error: template instance mir.reflection.Template!(Test2, "Value") error instantiating
./dlang/dmd-2.094.2/linux/bin64/../../src/druntime/import/core/internal/traits.d(199,21):        instantiated from here: F!(Test2)
./dmd-2.094.2/linux/bin64/../../src/phobos/std/meta.d(858,24):        instantiated from here: allSat!(ApplyRight, Test2)
./mir-core-1.1.59/mir-core/source/mir/algebraic.d(1485,20):        instantiated from here: allSatisfy!(ApplyRight, Test2)
./mir-core-1.1.59/mir-core/source/mir/algebraic.d(382,23):        ... (1 instantiations, -v to show) ...
./mir-core-1.1.59/mir-core/source/mir/algebraic.d(595,24):        instantiated from here: Variant!(typeof(null), Test2)
```

**Comment by 9il** [comment]
Thanks! fixed in v1.1.60

**Comment by skoppe** [comment]
Thanks man.
